test/VoucherHub.test.ts

```diff
@@ -968,6 +968,119 @@ describe('VoucherHub', function () {
         });
     });
 
+    describe('Drain voucher', function () {
+        let [voucherOwner1, anyone]: SignerWithAddress[] = [];
+        let voucherHub: VoucherHub;
+        let voucher: Voucher;
+        let voucherAddress: string;
+
+        beforeEach(async function () {
+            ({ voucherHub, voucherOwner1, anyone } = await loadFixture(deployFixture));
+            // Create voucher type
+            await voucherHubWithAssetEligibilityManagerSigner
+                .createVoucherType(description, duration)
+                .then((tx) => tx.wait());
+            // Create voucher
+            voucherAddress = await voucherHubWithVoucherManagerSigner
+                .createVoucher(voucherOwner1, voucherType, voucherValue)
+                .then((tx) => tx.wait())
+                .then(() => voucherHub.getVoucher(voucherOwner1));
+            voucher = Voucher__factory.connect(voucherAddress, anyone);
+        });
+
+        it('Should drain all funds of expired voucher', async function () {
+            const voucherHubRlcBalanceBefore = await iexecPocoInstance.balanceOf(voucherHubAddress);
+            // Expire voucher
+            const expirationDate = await voucher.getExpiration();
+            await time.setNextBlockTimestamp(expirationDate + 100n); // after expiration
```

Comment:
```suggestion
            await time.setNextBlockTimestamp(expirationDate); // after expiration
```

Reply:
(in both places)

Reply:
I had occurrences where the test would fail without it.
Let's how ti goes [Remove additional time of expiration](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/25/commits/81d7ee2dbf9fdb1da10bdcb3f3f86c0c37e8f74b)